CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -2190,6 +2190,15 @@ protected override void OnMouseDown(MouseButtonEventArgs e)
             Focus();
             OnMouseButton(e);
 
+            // We should only need to capture the left button exiting the browser
+            if (e.ChangedButton == MouseButton.Left && e.LeftButton == MouseButtonState.Pressed)
+            {
+                // Capture mouse so events are routed to the browser when it's outside of the control.
+                // This fixes selection issue https://github.com/cefsharp/CefSharp/issues/2870 and the issue with 
+                // no scrolling when on the right of a scroll bar: https://github.com/cefsharp/CefSharp/issues/2060
```

Comment:
> the issue with no scrolling when on the right of a scroll bar: https://github.com/cefsharp/CefSharp/issues/2060

This should probably reference be replaced with a reference to issue #2258 and make clear the limitation of the capture only working within the `WPF Window`.

Strictly speaking issue #2060 was resolved. Leaving the browser with the mouse down, releasing the mouse outside the browser and returning would still register as having the mouse down and scroll  the browser on mouse move. The fix for #2060 was limited and admittidly not particularl user friendly, hence #2258 was created.

Reply:
Revise that, I'm probably being too pedantic here. Having an extra link doesn't matter, the description not matching the actual issue was the part that was bugging me 😄 

Let's just go with something like below, focus on the intent of the code.

```c#
//Capture/Release Mouse to allow for scrollowing outside bounds of browser control (#2870, #2060).
//This doesn't capture the mouse outside of the WPF Window, see #2258  for further discussion on that topic.
```

Reply:
Sure, I've updated the comments :)